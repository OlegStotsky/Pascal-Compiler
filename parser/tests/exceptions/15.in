type
a = ^integer;

var
	b : integer;

begin
	b := a;
end;
