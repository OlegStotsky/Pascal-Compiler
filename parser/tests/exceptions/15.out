Error at line 8, column 9 : Error: a isn't variable
